crates/data_models/src/payments/payment_intent.rs

```diff
@@ -5,6 +5,7 @@ use time::PrimitiveDateTime;
 
 use crate::{errors, MerchantStorageScheme};
 const MAX_LIMIT: u32 = 100;
+const QUERY_LIMIT: u32 = 20;
```

Comment:
This const is created at 2 places...
can we try to reuse,
Also update the names to be somewhat LIMIT_V1 / LIMIT_V2 since I assume 100 is the older limit with GET request vs 20 is the newer with POST request